.github/workflows/docs/check-build-docs

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+./.github/workflows/docs/build-docs 1>out.txt 2>err.txt
+cat out.txt err.txt
+ERRS=`cat err.txt | grep -v "start-string without end-string"`
```

Comment:
I don't think this exclusion is necessary when building these docs.

Reply:
Will remove it.